Hooks/Keybindings.cs

```diff
@@ -36,7 +36,7 @@ static class Keybindings
 #nullable disable
     private static TryGetInputFlagLocalization_t TryGetInputFlagLocalization_Original;
     private static Harmony _harmony;
-    private static FastNativeDetour _detour;
+    private static NativeDetour _detour;
```

Comment:
Why MonoMod native detour? Does this even work? Use BepInEx's detouring API.
```suggestion
    private static INativeDetour _detour;
```